src/scripts/domain/game.ts

```
@@ -141,18 +140,33 @@ export class Game implements GameInfo, GameInteractable {
     return null;
   }
 
-  public performMove(pieceToMove: PieceInfo, move: Move) {
-    const pieceMoveInfo = this._moveInfos.find((mi) => mi.piece === pieceToMove);
+  public performMove(pieceToMoveInfo: PieceInfo, move: Move) {
+    const pieceMoveInfo = this._moveInfos.find((mi) => mi.piece === pieceToMoveInfo);
     if (!pieceMoveInfo) {
-      throw new Error('Unable to find moves for piece ' + pieceToMove);
+      throw new Error('Unable to find moves for piece ' + pieceToMoveInfo);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)